### DISCUSSION TITLE
Videos in NextJS

### DISCUSSION THREAD

**Discussion by RivkyB** [vercel/next.js#25771] [Help]
Anybody have experience with displaying videos? I am having a hard time finding any resources and I am a bit stuck. I tried using the native `<video>` but the video was super slow and choppy.
I am trying to achieve a video that plays when hovered and pauses when not hovered with looping. 
Any help I can get would be greatly appreciated!!

**Comment by ghost** [comment]
https://github.com/video-react/video-react
